### ISSUE TITLE
[Quests] bloodmage Drazial and Lynnore quests entering Blasted Lands

### ISSUE THREAD

**Issue by eduardojj** [dalaranwow/dalaran-wow#4840]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
After i've done the quests "Snickerfang Jowls", "The Decisive Striker" and "A Boar's vitality" i can't complete them although i've gathereth all to complete them. Strange thing is that the map shows the question mark saying it's complete.

**Expected behaviour**: Tell us what should happen instead
It should have let me complete the quests and receive the exp.

**Steps to reproduce the problem**:

1.Accept all quests given by the 2 bloodmages at the entrance to blasted lands. 
2. Complete at least one
3. Try to talk to the quest giver and complete the quest

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/
Bloodmage Drazial - https://wowgaming.altervista.org/aowow/?npc=7505
Bloodmage Lynnore - https://wowgaming.altervista.org/aowow/?npc=7506

Snickerfang jowls - https://wowgaming.altervista.org/aowow/?quest=2581
The decisive striker - https://wowgaming.altervista.org/aowow/?quest=2585
A boar's vitality - https://wowgaming.altervista.org/aowow/?quest=2583

**Include Screenshots from the issue if necessary**

![wowscrnshot_080217_185740](https://user-images.githubusercontent.com/30157305/28887305-9a686758-77b4-11e7-85fb-2e552326f489.jpg)

![inkedwowscrnshot_080217_190259_li](https://user-images.githubusercontent.com/30157305/28887546-6f60df44-77b5-11e7-9056-ba0571b54a6b.jpg)

**Comment by TeridanDW** [comment]
Hello @eduardojj. simply retaking the quests will solve your issue.

Note: This issue happens mostly when the items needed are not marked as "quest items".
